app/models/mixins/has_monitoring_manager_mixin.rb

```diff
@@ -0,0 +1,29 @@
+module HasMonitoringManagerMixin
+  extend ActiveSupport::Concern
+
+  included do
+    # TODO: why is this relation needed ? why is the inheritors relation no enough?
```

Comment:
I believe AR class methods like `has_one` don't create something inheritable (like defining a method).
That's why mixins do this under `included` to execute in each class instead of in the mixin. 

It obviously does work with class->class STI inheritance — dunno how exactly — but not mixins I think.
I may be wrong.